gr-utils/python/modtool/util_functions.py

```diff
@@ -136,3 +138,29 @@ def ask_yes_no(question, default):
         return default
     else:
         return not default
+
+class SequenceCompleter(object):
+    """ A simple completer function wrapper to be used with readline, e.g.
+    option_iterable = ("search", "seek", "destroy")
+    readline.set_completer(SequenceCompleter(option_iterable).completefunc)
+    """
+
+    def __init__(self, sequence):
+        self._seq = sequence
+        self._tmp_matches = []
+
+    def completefunc(self, text, state):
```

Comment:
Missing docstrings

Reply:
That doesn't move aside the fact that having the docstrings would be nice :slightly_smiling_face: